src/platform_impl/ios/window.rs

```diff
@@ -199,6 +199,11 @@ impl Inner {
         false
     }
 
+    pub fn is_visible(&self) -> bool {
+        warn!("`Window::is_visible` is ignored on iOS");
```

Comment:
Yeah, just looking at it by using `Option` you can clear state with `None` that visibility can't be determined.